app/views/components/modals/clients.ejs

```diff
@@ -41,7 +41,7 @@
                             <%=client.email%>
                         </td>
                         <td class="td-notif-image">
-                           <button onclick="toggleNotification(<%=client.id%>, <%=job.id%>, <%=client.is_recipient%>)"><img id="notification_bell" src="images/<%=client.is_recipient==true ? '' : 'no_' %>notification.svg" style="width: 20px; text-align: right;"></button>
+                           <a class="notif_button" id="<%=client.id%>"><img id="notification_bell<%=client.id%>" src="images/<%=client.is_recipient==true ? '' : 'no_' %>notification.svg" style="width: 20px; text-align: right;"></a>
```

Comment:
it will be a good practice to set the id of this something other than just plain <%=client.id%>, because that is just an integer, id needs to be unique, this is way too generic. name it something like `notif_button_<%=client.id%>` then in the function, use split to split the id to three parts, and you can grab the integer in index 2